doc/modules/multiclass.rst

```diff
@@ -30,21 +30,21 @@ by decomposing such problems into binary classification problems.
 
   - **Multioutput-multiclass classification** and **multi-task classification**
     means that a single estimator has to handle
-    several joint classification tasks. This is a generalization
+    several joint classification tasks. This is both a generalization
     of the multi-label classification task, where the set of classification
```

Comment:
I would say "which only considers binary classification". Currently the sentence is a bit awkward (what does "where" refer to?)